Cargo.toml

```diff
@@ -37,7 +37,7 @@ log = "0.4"
 lru = "0.6.6"
 num_cpus = "1.0"
 page_size = "0.4"
-rocksdb = "0.12"
+rocksdb = { version = "0.18.0", default-features = false }
```

Comment:
`0.12` just simply doesn't build on my machine. I updated to the latest `0.22.0` which did solve the build problem however it changed the interface around database iterators. So I went through the release notes and found when they made that change (19) and set it to one before that (18).